docs/management/admin/event-filters.asciidoc

```diff
@@ -42,7 +42,7 @@ image::images/event-filter.png[Add event filter flyout, 80%]
   .. `Description`: Enter a filter description (optional).
 . In the **Conditions** section, depending which page you're using to create the filter, either modify the pre-populated conditions or add new conditions to define how {elastic-sec} will filter events. Use these settings:
   .. `Select operating system`: Select the appropriate operating system.
-  .. Select which kind of event filter you'd like to create: added:[8.15.0,Coming to {serverless-full}.]
+  .. Select which kind of event filter you'd like to create: added:[8.15.0]
     * `Events`: Create a generic event filter that can match any event type. All matching events are excluded.
     * `Process Descendants`: Create a filter that suppresses the descendant activity of a specified process. Events from the matched process will be ingested, but events from its descendant processes will be excluded.
```

Comment:
Apply suggestion from [here](https://github.com/elastic/security-docs/pull/5672/files#r1715289035) (serverless) to ESS docs:
```suggestion
    * `Process Descendants`: Specify a process, and suppress the activity of its descendant processes. Events from the matched process will be ingested, but events from its descendant processes will be excluded.
```